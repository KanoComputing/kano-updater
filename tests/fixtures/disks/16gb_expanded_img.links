/dev/disk/by-label/boot -> ../../mmcblk0p1
/dev/disk/by-label/rootfs -> ../../mmcblk0p2
