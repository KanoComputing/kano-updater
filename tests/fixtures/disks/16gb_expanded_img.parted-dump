Model: SD SS16G (sd/mmc)
Disk /dev/mmcblk0: 31116288s
Sector size (logical/physical): 512B/512B
Partition Table: msdos
Disk Flags: 

Number  Start     End        Size       Type      File system  Flags
 1      8192s     3386718s   3378527s   primary   fat16        lba
 2      3386719s  31116287s  27729569s  primary   ext4        lba

