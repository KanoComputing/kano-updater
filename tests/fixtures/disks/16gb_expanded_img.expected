{
    "commands": [
        "parted /dev/mmcblk0 --script unit % resizepart 2 100 || parted /dev/mmcblk0 ---pretend-input-tty unit % resizepart 2 Yes 100",
        "partprobe /dev/mmcblk0",
        "resize2fs /dev/mmcblk0p2"
    ]
}
